rls/src/config.rs

```diff
@@ -103,6 +103,17 @@ impl<T> AsRef<T> for Inferrable<T> {
     }
 }
 
+/// Returns whether RLS was built using nightly channel.
+///
+/// It is very similar to what rust and rustfmt uses [1] - it relies on
+/// CFG_RELEASE_CHANNEL being set by Rust bootstrap.
+/// In case the env var is missing, we assume that we're built by Cargo and are
+/// using nightly since that's the only channel supported right now.
+/// [1]: https://github.com/rust-lang/rustfmt/blob/dfa94d150555da40780413d7f1a1378565208c99/src/config/config_type.rs#L53-L67
+pub fn is_nightly() -> bool {
```

Comment:
I think the name of this function, and the first line of the comment, could be changed to be more reflective of what it does? e.g. `build_supports_unstable()`

(calling it "is_nightly" when it returns true for "dev" feels odd...)

Reply:
I'd like to think of dev really as a special case of nightly but the changed name makes sense! Renamed now